lib/orderbook/OrderBook.ts

```diff
@@ -84,7 +85,7 @@ class OrderBook extends EventEmitter {
     return this.currencyInstances.keys();
   }
 
-  constructor(private logger: Logger, models: Models, public nomatching = false,
+  constructor(private logger: Logger, models: Models, private raidenDisabled: boolean, public nomatching = false,
```

Comment:
I believe we don't need to use this value at all because `SwapClientManager.get` will not return a swap client for the currency if the client is disabled.